curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-intermediate-oop-by-building-a-platformer-game/64cb4ebdc75b3a73a43da5ec.md

```diff
@@ -420,6 +420,9 @@ const startGame = () => {
 const showCheckpointScreen = (msg) => {
   checkpointScreen.style.display = "block";
   checkpointMessage.textContent = msg;
+  if (isCheckpointCollisionDetectionActive) {
```

Comment:
It looks like there is a misunderstanding of the issue.
You shouldn't give them the answer.
Also, we don't need to change all of the seed code here.
I would suggest reverting all of your changes to these files.

The issue are the tests for step 103 which can be found in the hints section `# --hints--`

The changes you need to make are to the tests themselves. 

The tests should accept any valid answer but right now they only accept one answer.

Once you have made updates to step 103's tests, make sure to run this locally and test your changes.

Hope that is clearer on what the task is 👍

Reply:
@jdwilkin4, Oh my bad, I misunderstood the issue. Let me fix.